samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction Finding Periodic Advertising Locator
+#########################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The direction finding connectionless locator sample application demonstrates Bluetooth LE Direction Finding reception.
```

Comment:
Should the name of the sample be connectionless locator or periodic advertising locator?

Reply:
it should be aligned with connectionless beacon sample, so word "connectionless" instead of "periodic advertising"